client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/task-helper.tsx

```diff
@@ -50,16 +49,15 @@ export function getEnhancedTasks(
 	submit: NavigationControls[ 'submit' ],
 	displayGlobalStylesWarning: boolean,
 	globalStylesMinimumPlan: string,
-	setShowPlansModal: Dispatch< SetStateAction< boolean > >,
 	queryClient: QueryClient,
-	goToStep?: NavigationControls[ 'goToStep' ],
 	flow: string | null = '',
 	isEmailVerified = false,
 	checklistStatuses: LaunchpadStatuses = {},
+	setShowConfirmModal: () => void,
+	goToStep?: NavigationControls[ 'goToStep' ],
```

Comment:
Linting required some changes that otherwise aren't needed for the PR.
  - I adjusted order to resolve 'required props cannot follow optional props'
  - I adjusted the type on setShowConfirmModal to resolve 'no empty arrow functions'.